ankaios_sdk/ankaios.py

```diff
@@ -297,6 +308,10 @@ def _read_from_control_interface(self) -> None:
                     if next_byte[0] & MOST_SIGNIFICANT_BIT_MASK == 0:
                         break
                 if not varint_buffer:  # pragma: no cover
+                    self.logger.error(
+                        "Empty buffer. Is the agent still there?"
```

Comment:
Let's be a bit more clear:
```suggestion
                        "Nothing to read from the input fifo pipe. Is the agent still there?"
```